Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
@cassianotartari manually flipped the colors, and then the default, deprecated path flips them again, which means if you upgrade… you will get the same as before, so no BC-break, right?